libs/core/mappers/query-params.mapper.ts

```diff
@@ -0,0 +1,60 @@
+import { AnimeTypeDto } from '../dtos/anime-type.dto';
+import { QueryParamsDto } from '../dtos/query-params.dto';
+import { QueryParams } from '../models/query-params';
+
+import { AnimeSortMapper } from './anime-sort.mapper';
+import { AnimeTypeMapper } from './anime-type.mapper';
+
+export namespace QueryParamsMapper {
+
+	/**
+	 * Map query params dto to model.
+	 * @param params - Query params.
+	 * @returns Model with query params data for UI.
+	 */
+	export function fromDto(params: QueryParamsDto): QueryParams {
+		const queryParams: QueryParams = {
+			offset: 0,
+			limit: 25,
+			search: '',
+			type: [],
+			ordering: { active: '', direction: '' },
+		};
+		if (params.offset) {
+			queryParams.offset = params.offset;
+		}
```

Comment:
An example for `offset` how it can be simplified

```suggestion
		const queryParams: QueryParams = {
			offset: params.offset ? params.offset : DEFAULT_OFFSET,
                }
```